SSA/Core/ErasedContext.lean

```diff
@@ -198,6 +199,11 @@ theorem Valuation.snoc_toSnoc {Γ : Ctxt Ty} {t t' : Ty} (s : Γ.Valuation) (x :
     (v : Γ.Var t') : (s.snoc x) v.toSnoc = s v := by
   simp [Ctxt.Valuation.snoc]
 
+/-- Build valuation from a list of values of types `types`. -/
+def Valuation.ofList {types : List Ty} : HVector toType types → Valuation (Ctxt.ofList types)
```

Comment:
I feel like this should be `Valuation.ofHVector`, but I guess arguments can be made for both

Reply:
Renamed as requested